components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/AuthenticationService.java

```diff
@@ -274,9 +275,8 @@ private String getErrorMessage(AuthServiceResponseWrapper response) throws AuthS
         return queryParams.get(AuthServiceConstants.AUTH_FAILURE_MSG_PARAM);
     }
 
-    private List<AuthenticatorData> getAuthenticatorBasicData(String authenticatorList,
-                                                              List<AuthenticatorData> authInitiationData)
-            throws AuthServiceException {
+    private List<AuthenticatorData> getAuthenticatorBasicData(AuthServiceRequestWrapper request,
+            String authenticatorList, List<AuthenticatorData> authInitiationData) throws AuthServiceException {
```

Comment:
Why not just pass the tenant domain here

Reply:
This is used in loading the context. So we need to resolve the tenant domain from the request.

Reply:
yes we can just pass the tenant domain from the request here without passing the request itself